src/ttt/game_rules.clj

```diff
@@ -0,0 +1,57 @@
+(ns ttt.game-rules
+  (:require [clojure.math.numeric-tower :as math]))
+
+(defn- get-board-size
+  [board]
+  (math/sqrt (count board)))
+
+(defn- create-columns
+  [board]
+ (apply mapv vector (partition (get-board-size board) board)))
+
+(defn- create-rows
```

Comment:
This might be good to call when printing the game board.  If so, this would have to be promoted to a public method and test(-driven).